doc.go

```diff
@@ -111,6 +111,7 @@ The following command line commands are provided by lf:
 	cmd-word                 (default '<a-f>')
 	cmd-word-back            (default '<a-b>')
 	cmd-delete-word          (default '<a-d>')
+	cmd-delete-word-back     (default '<a-backspace> and <a-backspace2>')
```

Comment:
I think you made a small typo:

```suggestion
	cmd-delete-word-back     (default '<a-backspace>' and '<a-backspace2>')
```